example/16_gemm_multi_d_multi_reduces/gemm_max_xdl_int8.cpp

```diff
@@ -0,0 +1,166 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2018-2022, Advanced Micro Devices, Inc. All rights reserved.
+
+#include "gemm_reduce_xdl_common.hpp"
+
+#include "ck/library/reference_tensor_operation/cpu/reference_gemm.hpp"
+#include "ck/tensor_operation/gpu/device/device_gemm_multiple_d_multiple_r_xdl_cshuffle.hpp"
+#include "ck/tensor_operation/gpu/device/gemm_specialization.hpp"
+
+using ADataType         = INT8;
+using BDataType         = INT8;
+using GemmAccDataType   = INT32;
+using CShuffleDataType  = INT32;
+using DsDataType        = ck::Tuple<>;
+using EDataType         = INT32;
```

Comment:
Sync the style withgemm_add_addsquare_xdl_int8.cpp. using EDataType = INT8;

Reply:
Done [88ce32a](https://github.com/ROCmSoftwarePlatform/composable_kernel/pull/368/commits/88ce32ac59516d2a5151e12e6995d5290685c83a)